crates/holochain/src/core/ribosome/host_fn/zome_info.rs

```diff
@@ -16,12 +16,17 @@ pub fn zome_info(
         HostFnAccess {
             bindings_deterministic: Permission::Allow,
             ..
-        } => ribosome
-            .zome_info(call_context.zome.clone())
-            .map_err(|e| match e {
-                RibosomeError::WasmRuntimeError(wasm_error) => wasm_error,
-                other_error => wasm_error!(WasmErrorInner::Host(other_error.to_string())).into(),
-            }),
+        } => {
+            let f = ribosome.zome_info(call_context.zome.clone());
+            tokio_helper::block_on(f, std::time::Duration::from_secs(60))
```

Comment:
This is the only place where blocking needed to be reintroduced. This is offset by several places where a blocking thread previously needed to be spawned to accommodate a long-running sync task, and now those tasks are async and can run normally in tokio.